### PR TITLE
Finalized UI, Places and Insights Functionality

### PR DESCRIPTION
**Pull request by pollchowdhury**
Merged Mahnoor's code and My code to main

### CODE REVIEW

**Comment by pollchowdhury** [comment]
The Updates can be merged to main automatically

**Comment by jchunduri35** [comment]
Finalized UI implemented

File: Proxyserver.js [hunk 1]
```diff
@@ -68,6 +70,8 @@ app.get("/oilchange", async (req, res) => {
         foursquare_results.results[1].location.formatted_address,
       rating: foursquare_results.results[1].rating / 2,
       review_count: foursquare_results.results[1].stats.total_ratings,
+      lat: foursquare_results.results[1].geocodes.main.latitude,
+      lng: foursquare_results.results[1].geocodes.main.longitude,
       recommend: "yes",
     });
     // storing nearby places with the yelp api results
```
Comment (andreacuc) [inline Proxyserver.js:79]: try implement recursion to create cleaner code as there is a ton of repetition though-out this file

File: src/Components/Automotive/Automative.js [hunk 2]
```diff
@@ -19,17 +19,21 @@ function AutomativeDropdown() {
   const handleChange = (event) => {
     setService(event.target.value);
   };
```
Comment (andreacuc) [inline src/Components/Automotive/Automative.js]: improve on in-code documentation explaining each component why it may be commented out

**Review by andreacuc — COMMENTED** [review]
code-review

**Pull request merged**